system-addon/content-src/styles/activity-stream-linux.scss

```diff
@@ -1,2 +1,8 @@
 /* This is the linux variant */
+
+$os-arrow-offset-end: 6px;
+$os-arrow-width: 20px;
+$os-search-focus-shadow-radius: 3px;
+$os-section-info-option-image: 'chrome://global/skin/arrow/panelarrow-vertical-themed.svg';
```

Comment:
The arrow stuff is directly referring to the panelarrow. These variable names make it seem like they're unrelated.